js-simulation/package.json

```diff
@@ -5,11 +5,13 @@
   "type": "module",
   "main": "target/bundle",
   "dependencies": {
-    "@gatling.io/core": "0.0.0",
-    "@gatling.io/http": "0.0.0"
+    "@gatling.io/core": "../tmp/core/package",
+    "@gatling.io/http": "../tmp/http/package",
+    "@gatling.io/mqtt": "../tmp/mqtt/package",
+    "uuid": "11.1.0"
   },
   "devDependencies": {
-    "@gatling.io/cli": "0.0.0",
+    "@gatling.io/cli": "../tmp/cli/package",
```

Comment:
Is this change intended?

Reply:
Yes, it simplifies the usage of npm link/install...